app/views/users/decks.erb

```diff
@@ -11,6 +13,6 @@
     </form>
     <form action="/decks/4" method="get">
       <button type="submit">Git Commands</button>
-    </form>
+    </form> -->
   </div>
 </div>
```

Comment:
:shipit:  <-- This is ShipIt Squirrel. Github uses him to denote that code is ready to be deployed. I think everyone should employ ShipIt Squirrel you do it with \:shipit followed immediately by another :
for some reason I can't do the escape character on the second colon.